native-sql-engine/core/src/main/scala/com/intel/oap/extension/ColumnarOverrides.scala

```diff
@@ -44,7 +44,7 @@ case class ColumnarPreOverrides() extends Rule[SparkPlan] {
   var isSupportAdaptive: Boolean = true
 
   def replaceWithColumnarPlan(plan: SparkPlan): SparkPlan = plan match {
-    case RowGuard(child: CustomShuffleReaderExec) =>
+    case RowGuard(child: AQEShuffleReadExec) =>
```

Comment:
In spark 3.2, CustomShuffleReaderExec is renamed to AQEShuffleReadExec.

Reply:
ShufflePartitionUtils.scala also imported CustomShuffleReaderExec, so also needs to be fixed.